tests/entities_column_value_readonly_tests.py

```diff
@@ -0,0 +1,93 @@
+import json
+from nose.tools import ok_, eq_, raises
+
+from moncli import entities as en, error as e
+from moncli.enums import *
+
+def test_should_create_column_value_with_empty_list_value_using_no_input_api_data():
+    
+    # Arrange
+    id = 'file1'
+    title = "file"
+    column_type = ColumnType.file
+    value = []
+
+    # Act
+    column_value = en.cv.create_column_value(column_type, id=id, title=title,value=value)
+
+    # Assert
+    eq_(column_value.value,[])
+
+
+def test_should_create_column_value_with_an_item_id_list_value_using_input_api_data():
```

Comment:
Please name these according to the type that is being tested.

Ex. test_should_create_file_column... etc.